### PR TITLE
Add migration guide

### PR DESCRIPTION
**Pull request by rhames07**
## Description
Add migration guide from v2 to v3.

### CODE REVIEW

File: MIGRATION_GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+The way we handle exceptions has changed. Now it's easier to identify if the error is from some API catching the `MPApiException` and accessing it's status code and content.
+
+```php
+catch (MPApiException $e) {
+    echo $e->getApiResponse()->getStatusCode();
+    echo $e->getApiResponse()->getContent();
+}
+```
+
+And not API errors can be catched by using the standart Exception.
```
Comment (gdeandradero) [inline MIGRATION_GUIDE.md]:
```suggestion
And not API errors can be catched by using the standard Exception.
```

File: MIGRATION_GUIDE.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+```php
+catch (Exception $e) {
+    echo $e->getMessage();
+}
+```
+
+### Step 5: Review the Documentation
+The new version of the MercadoPago PHP SDK comes with more comprehensive and up-to-date documentation. We recommend that you review the documentation to understand all the changes and take advantage of the new available features.
+
+- [SDKs documentation](https://www.mercadopago.com.br/developers/en/docs/sdks-library/server-side).
```
Comment (gdeandradero) [inline MIGRATION_GUIDE.md]:
```suggestion
- [SDKs documentation](https://www.mercadopago.com/developers/en/docs/sdks-library/server-side).
```

File: MIGRATION_GUIDE.md (hunk 1 above)
Comment (rhames07) [reply MIGRATION_GUIDE.md]: Feito.

File: MIGRATION_GUIDE.md (hunk 2 above)
Comment (rhames07) [reply MIGRATION_GUIDE.md]: Feito.

**Review by lucmantovani — APPROVED** [review]

**Pull request merged**